client/src/components/layouts/Booking/exceptedDays.js

```diff
@@ -0,0 +1,22 @@
+// if the user have covid symptoms, the reservation will be available after 14 days
+const covidBlockedDays = 14;
+const today = new Date();
+
+export default {
+  today: new Date(),
+  blockedWeeks: new Date(
+    today.getFullYear(),
+    today.getMonth(),
+    today.getDate() + covidBlockedDays,
```

Comment:
This looks a little dangerous to me, for those situations where we end up setting something like
`blockedWeeks: new Date(2020, 07, 35)`.

I tested in Chrome and it does work as expected, but this is running on the client and it could be a source of really tricky bugs if not all browsers handle this in the same way.

See on MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date `Note: Parsing of date strings with the Date constructor (and Date.parse, they are equivalent) is strongly discouraged due to browser differences and inconsistencies.`

It's better to use moment on the frontend. It's safer to use native dates on the backend because you're more in control of the environment.

Reply:
I had to use a new Date with the dates because the calendar that we use accepts only dates as an instance of Date, so if I remove it, I got this warning.
![Screenshot from 2020-07-21 22-48-10](https://user-images.githubusercontent.com/54964739/88113053-b5573100-cbb9-11ea-971a-25d3cb624fe5.png)